### PR TITLE
Add bgerm handler

### PR DESCRIPTION
**Pull request by jrmlhermitte**

### CODE REVIEW

**Comment by jrmlhermitte** [comment]
needs tests (currently tested with data given by Tom)

awaiting simulation

**Comment by jrmlhermitte** [comment]
test added. cleaning up

NOTE: Still need to figure out expected endianness

**Comment by tacaswell** [comment]
Can you get travis set up for this project as well?

File: pygerm/tests/test_hdf5germ.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# some setup
```
Comment (jrmlhermitte) [inline pygerm/tests/test_hdf5germ.py:1]: oops need to remove this. will later. it's unclear to me if we want to turn this into a test for the older hdf5 format

**Comment by jrmlhermitte** [comment]
@tacaswell sure will do

File: pygerm/tests/test_hdf5germ.py (hunk 1 above)
Comment (tacaswell) [reply pygerm/tests/test_hdf5germ.py:1]: The long-term format is not clear.  

Currently this is a R&D project and the binary format is what the detector group is used to working with.  Long term (when this passes to being a production detector at XPD) we may want to go back to the hdf5 format (at it plays much nicer with the data broker and gives fast column-wise access).

**Comment by jrmlhermitte** [comment]
@tacaswell which versions do we want to support? currently this is py3.6+ (uses f-strings)

**Comment by tacaswell** [comment]
This is mostly internal-usage, 3.6+ is fine.

The reduction module should support 2.7+, and we should not import the other modules by default.

**Comment by tacaswell** [comment]
Closing this as I merged it into #4 

Thanks @jrmlhermitte

**Pull request closed without merging**